agora_analytica/data/interpolation/combine.py

```diff
@@ -5,59 +5,79 @@
 import os.path
 import json
 from ..utils import instance_path
+from ..scrape import hae_dataa
 
 instancepath = instance_path() / "yle_scrape"
 instancepath.mkdir(exist_ok=True)
 
-IMAGE_BASE_URL = "https://ehdokaskone.yle.webscale.fi/"
+#instancepath = os.path.join(instance_path(), "scraped")
+
+# load in parties and constituents data
+# if data is missing ask if scraping is wanted to be performed
+while True:
+    try:
+        with open(os.path.join(instancepath, f"parties.json"), "r", encoding='utf-8') as json_file:
+            parties = json.load(json_file)
+
+        with open(os.path.join(instancepath, f"constituencies.json"), "r", encoding='utf-8') as json_file:
+            constituencies = json.load(json_file)
+        break
+    except FileNotFoundError:
+        text = input("Seems like you are missing some data. Do you want to scrape now, THIS MIGHT TAKE UP TO 35 minutes Y/N? (case sensitive)") 
+        if (text.upper() == "Y"):
+            hae_dataa()
+        else:
+            raise Exception("You choose to not scrape: Missing data can't build")
+
+# load the scraped data to its own data frame
+df_scraped = pd.DataFrame(columns=['first_name', 'last_name', 'election_number', 'image', 'election_promise_1', 'party', 'constituency'])
+i = 1
+while i <= 3000:
+    try:
+        with open(os.path.join(instancepath, f"candidate" + str(i) + ".json"), "r", encoding='utf-8') as json_file:
```

Comment:
Voit kerätä kaikki täsmäävät tiedostot [globilla](https://docs.python.org/3/library/glob.html):
```python
from glob import glob

for filename in glob(f"{instancepath}/candidate*.json"):
    with open(filename, "r") as json_file:
```

Reply:
Tätä en saa toimimaan.
Laitoin tohon for ja with rivien väliin print komennon ja se suoritetaan vaan kerran
eli toi for looppi löytää vain yhden tiedoston.